.coveragerc

```diff
@@ -441,6 +441,7 @@ omit =
     homeassistant/components/ping/device_tracker.py
     homeassistant/components/pioneer/media_player.py
     homeassistant/components/pjlink/media_player.py
+    homeassistant/components/plaato/*
```

Comment:
Please add a test for the config flow and do not exclude the config flow file from coverage, so list each file untested file individually.

Reply:
Nevermind, you're using the webhook flow